installers/helm/templates/NOTES.txt

```diff
@@ -0,0 +1,23 @@
+Thank you for installing the Crunchy PostgreSQL Operator v{{ .Chart.AppVersion }}!
+
+  .-''''-. _    
+ ('    '  '0)-/)
+ '..____..:    \._
+   \u  u (        '-..------._
+   |     /      :   '.        '--.
+  .nn_nn/ (      :   '            '\
+ ( '' '' /      ;     .             \
+  ''----' "\          :            : '.
+         .'/                           '.
+        / /                             '.
+       /_|       )                     .\|
+         |      /\                     . '
+         '--.__|  '--._  ,            /
+                      /'-,          .'
+                     /   |        _.' 
+                    (____\       /    
+                          \      \    
+                           '-'-'-'
+
```

Comment:
should we add a copyright for the ascii hippo? Do we have a crunchy ascii hippo?

Reply:
Did you create it?

I don’t think we currently have ascii art of the Crunchy hippo

Reply:
@jkatz no - i think Andrew found it somewhere on the webs